Backend/app/spotify_electron/user/user_controller.py

```diff
@@ -325,7 +367,9 @@ def get_user_relevant_playlists(name: str) -> Response:
     try:
         playlists = base_user_service.get_user_relevant_playlists(name)
         playlists_json = json_converter_utils.get_json_from_model(playlists)
-        return Response(playlists_json, media_type="application/json", status_code=HTTP_200_OK)
+        return Response(
```

Comment:
This is the other example I saw related to line length comment above

Reply:
Oh actually, it turns out I have Black's default line length which is 88, and it seems Black doesn't support reading configuration from ruff.toml, only pyproject.toml? I'll change my Black default line settings to 95 which is what you have on the ruff.toml